Tools/packages.config

```diff
@@ -7,7 +7,6 @@
   <package id="NUnit.Extension.NUnitV2ResultWriter" version="3.6.0" targetFramework="net461" />
   <package id="NUnit.Extension.TeamCityEventListener" version="1.0.6" targetFramework="net461" />
   <package id="NUnit.Extension.VSProjectLoader" version="3.8.0" targetFramework="net461" />
-  <package id="CSharpAsyncGenerator.CommandLine" version="0.14.0" targetFramework="net461" />
   <package id="vswhere" version="2.1.4" targetFramework="net461" />
   <package id="GitReleaseManager" version="0.7.0" targetFramework="net461" />
```

Comment:
This one is referenced [here](https://github.com/nhibernate/nhibernate-core/blob/master/ReleaseProcedure.txt#L6):

> Create a draft release in Github with GitReleaseManager. If you have used the NHibernate build menu, it should be available in Tools\gitreleasemanage.x.x.x\

So if running the build menu does no more trigger a fetch of this package, this sentence above will no more hold true.

Reply:
Thanks for the references, I've moved it to `packages.csproj` and updated the release procedure.